src/Message/TransactionStatusRequest.php

```diff
@@ -42,4 +57,106 @@ public function sendData($data): ResponseInterface
             $this->getResponse()->getStatusCode()
         );
     }
+
+    /**
+     * Use the data sent by Icepay in the post back to check the status.
+     * This is necessary because Icepay has a delay in their backend if you request the status immediately after the signal.
+     *
+     * @see http://docs2.icepay.com/payment-process/handling-the-postback/postback-sample/
+     *
+     * @return TransactionStatusResponse|bool false when the data is is not sent or not correct
+     *
+     * @throws PostBackException by call to self::validateSecurityHashMatch()
+     */
+    private function getTransactionStatusFromPostBack()
```

Comment:
- Can we return `null` instead of `false` if we cannot read the postback response?
- Better idea to call it `getTransactionStatusResponseFromPostbackData`?

Reply:
Done, and at a later moment